derive_builder_no_std_tests/Cargo.toml

```
@@ -0,0 +1,11 @@
+[package]
+name = "derive_builder_no_std_tests"
+version = "0.1.0"
+authors = ["Andrey Kutejko <andy128k@gmail.com>"]
+edition = "2018"
```

```suggestion
edition = "2018"
publish = false
```

This will prevent the package from being published accidentally ([source](https://doc.rust-lang.org/cargo/reference/manifest.html#the-publish-field)).